### ISSUE TITLE
Dataframe normalize functions

### ISSUE THREAD

**Issue by yannikschaelte** [PEtab-dev/PEtab#277]
Should we have "normalize" functions for each of the tables which fill in all optional columns with the default values (maybe returning a copy?)?

**Comment by dweindl** [comment]
Is it only about having those functions, or about not having to check for presence of the respective columns elsewhere?

In the latter case, I am not so sure. I would like to avoid creating copies or going through the whole DataFrame in every other function just for convenience. Doing it only once, and assuming everything is filled in in other parts of the library, somehow removes the point of optional columns...

**Comment by yannikschaelte** [comment]
only about having those functions (agreeing with the latter point): I do not want to repeat looking up default values for each tool anew.

**Comment by dweindl** [comment]
Ok. Fine with me.

**Comment by yannikschaelte** [comment]
:+1:

**Comment by yannikschaelte** [comment]
now exists for parameter df, which is the critical one. not needed for others atm.
